### PR TITLE
add a decorator for cross_entropy for 'target' matchting 'label'

### PR DESCRIPTION
**Pull request by hd9568**
### PR Category

User Experience 

### PR Types

Security
### Description

add a decorator for cross_entropy for 'target' matchting 'label'

### CODE REVIEW

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -2679,7 +2679,7 @@ def softmax_with_cross_entropy(
         axis,
     )
 
-
+@ParamAliasDecorator({"target": ["label"]})
```
Comment (zhwesky2010) [inline python/paddle/nn/functional/loss.py]: 这个加下测试吧，找一下softmax_with_cross_entropy的单测，测一下 softmax_with_cross_entropy(target=) 的用法

**Comment by zhwesky2010** [comment]
还需要实现一个 `paddle.long`，是 `paddle.int64` 的别名

File: python/paddle/nn/functional/loss.py (hunk 1 above)
Comment (hd9568) [reply python/paddle/nn/functional/loss.py]: done

**Review by zhwesky2010 — DISMISSED** [review]
LGTM

**Review by zhwesky2010 — APPROVED** [review]
LGTM
